src/main.c

```diff
@@ -356,9 +364,16 @@ static struct {
     .tcp_reuseport = 0,
     .ssl_zerocopy = 0,
     .ocsp_updater = {.capacity = H2O_DEFAULT_OCSP_UPDATER_MAX_THREADS},
+    .bp = {
+         .epoll_bp_usecs = 0,
+         .epoll_bp_budget = 0,
+         .epoll_bp_prefer = false,
+         .epoll_bp_changed = false,
+         .epoll_nonblock = false,
+     }
 };
 
-static __thread size_t thread_index;
+static __thread size_t thread_index = SIZE_MAX;
```

Comment:
Do we need this change?

Reply:
No, reverted in cff14365dc3baee042cc638995fad5b1c01868cb